handbook/filesystem.md

```diff
@@ -0,0 +1,80 @@
+## EasyEngine on the Filesystem
+
+All files created by EasyEngine are stored at `/opt/easyengine`
+
+## Sites
+
+By default, all EasyEngine sites are created at `/opt/easyengine/sites`. Configuration and logs of individual site's NGINX, PHP, Postfix are stored in `/opt/easyengine/sites/example.com`. 
+
+Checkout the documentation on [a site's filesystem structure](https://github.com/EasyEngine/docs/blob/master/handbook/site-structure.md) to know more about it.
+
+## Services
+
+There are some containers in EasyEngine which serves multiple sites. These are called services. We have following services in EasyEngine:
+
+1. MariaDB
+2. Nginx Proxy
+3. Cron Scheduler
+
+Files of these services can be found at `/opt/easyengine/services`
+
+### MariaDB
+#### Logs
+Logs of MariaDB can be found at `/opt/easyengine/services/mariadb/logs`.
+
+#### Config
+Config of MariaDB can be found at `/opt/easyengine/services/mariadb/config`.
+You can add custom config at `/opt/easyengine/services/mariadb/conf/mariadb.conf.d/`
+
+#### Data
+Data of MariaDB can be found at `/opt/easyengine/services/mariadb/data`.
+
+### Nginx Proxy
+#### Logs
+Logs of Nginx Proxy can be found at `/opt/easyengine/services/nginx-proxy/logs`.
+
+#### Config
+Config of Nginx Proxy can be found at `/opt/easyengine/services/nginx-proxy/conf.d`.
+
+One part of Nginx Proxy is autogenerated from [go template](https://github.com/EasyEngine/dockerfiles/blob/master/nginx-proxy/nginx.tmpl) and can be found at `/opt/easyengine/services/nginx-proxy/conf.d/default.conf`. Do not modify this file as gets regenerated whenever a container starts/stops.
```

Comment:
This part can be improved. Mainly mentioning why the default.conf is being regenerated, even if we don't give detailed description, at least one/two line should be there.

Reply:
_Do not modify this file as **it** gets regenerated whenever a container starts/stops._